crates/oxc_cli/src/result.rs

```diff
@@ -18,6 +19,12 @@ impl Termination for CliRunResult {
                 println!("Path {paths:?} does not exist.");
                 ExitCode::from(1)
             }
+            Self::MaxWarningsExceeded { number_of_warnings, max_warnings } => {
+                println!(
+                    "Exceeded --max-warnings value of {max_warnings}. Found {number_of_warnings} warning(s)."
+                );
```

Comment:
Just went with what made sense. Not sure if the other outputs are based on some reference material or not, happy to change

Reply:
This is fine.